keras_hub/src/models/differential_binarization/differential_binarization_ocr.py

```diff
@@ -0,0 +1,113 @@
+import keras
+from keras import layers
+
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.differential_binarization.differential_binarization_backbone import (
+    DifferentialBinarizationBackbone,
+)
+from keras_hub.src.models.differential_binarization.differential_binarization_preprocessor import (
+    DifferentialBinarizationPreprocessor,
+)
+from keras_hub.src.models.differential_binarization.losses import DBLoss
+from keras_hub.src.models.image_segmenter import ImageSegmenter
+
+
+@keras_hub_export("keras_hub.models.DifferentialBinarizationOCR")
+class DifferentialBinarizationOCR(ImageSegmenter):
+    """
+    A Keras model implementing the Differential Binarization
+    architecture for scene text detection, described in
+    [Real-time Scene Text Detection with Differentiable Binarization](
+    https://arxiv.org/abs/1911.08947).
+
+    Args:
+        backbone: A `keras_hub.models.DifferentialBinarizationBackbone`
+            instance.
+        preprocessor: `None`, a `keras_hub.models.Preprocessor` instance,
+            a `keras.Layer` instance, or a callable. If `None` no preprocessing
+            will be applied to the inputs.
+
+    Examples:
+    ```python
+    input_data = np.ones(shape=(8, 224, 224, 3))
+
+    image_encoder = keras_hub.models.ResNetBackbone.from_preset(
+        "resnet_vd_50_imagenet"
+    )
+    backbone = keras_hub.models.DifferentialBinarizationBackbone(image_encoder)
+    detector = keras_hub.models.DifferentialBinarizationOCR(
+        backbone=backbone
+    )
+
+    detector(input_data)
```

Comment:
What does the output of the task look like?

Reply:
We get the probability map, threshold map and binary map output in the last dimension from the model. I've added some documentation here.